### ISSUE TITLE
Misleading error when doing a `glide install` or `glide get` on a committed vendor dir

### ISSUE THREAD

**Issue by technosophos** [Masterminds/glide#285]
This issue surfaced as part of #275.

Something is wonky about the error messages reported on packages in `vendor/` that are checked into VCS. I think we should be able to better detect this case and emit a clearer error message.

```
⇒  git clone https://github.com/nlamirault/abraracourcix
Cloning into 'abraracourcix'...
remote: Counting objects: 1985, done.
remote: Compressing objects: 100% (91/91), done.
remote: Total 1985 (delta 39), reused 0 (delta 0), pack-reused 1884
Receiving objects: 100% (1985/1985), 6.91 MiB | 757.00 KiB/s, done.
Resolving deltas: 100% (492/492), done.
Checking connectivity... done.
☃ @technosophos.local ~/Code/Go/src/github.com «/Users/mbutcher/Code»
⇒  cd abraracourcix
☃ @technosophos.local ~/Code/Go/src/github.com/abraracourcix «/Users/mbutcher/Code» ‹master›
⇒  ls
API.md                Dockerfile            README.md             api                   glide.yaml            storage
CONTRIBUTING.md       LICENSE               abraracourcix.go      circle.yml            io                    vendor
ChangeLog.md          Makefile              abraracourcix_test.go contrib               logging               version
☃ @technosophos.local ~/Code/Go/src/github.com/abraracourcix «/Users/mbutcher/Code» ‹master›
⇒  glide up
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/garyburd/redigo.
[INFO] Fetching updates for github.com/hashicorp/logutils.
[INFO] Fetching updates for github.com/labstack/echo.
[INFO] Fetching updates for github.com/boltdb/bolt.
[INFO] Fetching updates for github.com/syndtr/goleveldb.
[INFO] Fetching updates for gopkg.in/mgo.v2.
[INFO] Resolving imports
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/api (1)
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/io (1)
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/logging (1)
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/storage (1)
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/version (1)
[INFO] Not found in vendor/: github.com/nlamirault/abraracourcix/api/v1 (1)
[ERROR] Error scanning github.com/labstack/echo/middleware: open /Users/mbutcher/Code/Go/src/github.com/abraracourcix/vendor/github.com/labstack/echo/middleware: no such file or directory
[ERROR] Error scanning github.com/garyburd/redigo/redis: open /Users/mbutcher/Code/Go/src/github.com/abraracourcix/vendor/github.com/garyburd/redigo/redis: no such file or directory
[ERROR] Error scanning github.com/syndtr/goleveldb/leveldb: open /Users/mbutcher/Code/Go/src/github.com/abraracourcix/vendor/github.com/syndtr/goleveldb/leveldb: no such file or directory
[ERROR] Error scanning gopkg.in/mgo.v2/bson: open /Users/mbutcher/Code/Go/src/github.com/abraracourcix/vendor/gopkg.in/mgo.v2/bson: no such file or directory
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/nlamirault/abraracourcix.
[INFO] Setting references for remaining imports
[INFO] Project relies on 7 dependencies.
An Error has occured
```

**Comment by mattfarina** [comment]
Can you please look at the pull request on #286

**Comment by mattfarina** [comment]
Fixed in #286.
